ext/rp/pico.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022, Andrey Kunitsyn
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#pragma once
+
+/**
+ * Pico SDK compatibility for tinyusb
+ */
+
+
+#include <stdint.h>
+#include <assert.h>
+
+#define TUD_OPT_RP2040_USB_DEVICE_ENUMERATION_FIX 0
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+#undef assert
+#define assert(X) do { } while(false)
```

Comment:
```suggestion
#define assert(X) modm_assert(0, "pico", __FILE__ ":" MODM_STRINGIFY(__LINE__) " -> \"" #X "\"")
```
something like that, I've used that for the [TinyUSB assert macro](https://github.com/modm-io/modm/blob/develop/ext/hathach/tusb_config.h.in#L29-L32) and it's pretty good at giving you enough information to pin down the problem (the second long string is only printed in debug mode to keep firmware size down).